app/src/main/java/com/andela/art/securitydashboard/presentation/NfcSecurityDashboardActivity.java

```diff
@@ -274,8 +274,8 @@ public void setAccountDetails(String email, String name, String photo) {
 
     @Override
     public void displayErrorMessage(Throwable error) {
-        String message = error.getMessage().toString();
-        toast = Toast.makeText(this, message, Toast.LENGTH_LONG);
+        String message = "The asset serial number is not available. " + error.getMessage();
```

Comment:
Can you use "The asset is not available." instead. The test checks for "The asset is not available."

Reply:
Done

Reply:
LGTM